mattermost/mattermost-webapp
# E2E for sidebar and link navigation of system admin console

#### Summary
E2E for sidebar and link navigation of system admin console

Test steps:
1. Visit the admin console directly via URL and verify that it matches content's header
2. Go to other admin URL, then click the section on the admin sidebar.  Verify that it matches URL and content's header

Note:
- Specs are separated for those in enterprise and non-enterprise features.
- Pre-packaged plugins will be tested separately

#### Ticket Link
none
